cmd/minikube/cmd/start_flags.go

```diff
@@ -164,6 +166,8 @@ func initMinikubeFlags() {
 	startCmd.Flags().StringP(network, "", "", "network to run minikube with. Now it is used by docker/podman and KVM drivers. If left empty, minikube will create a new network.")
 	startCmd.Flags().StringVarP(&outputFormat, "output", "o", "text", "Format to print stdout in. Options include: [text,json]")
 	startCmd.Flags().StringP(trace, "", "", "Send trace events. Options include: [gcp]")
+	startCmd.Flags().Int(extraDisks, 0, "Number of extra disks created and attached to the minikube VM.")
+	startCmd.Flags().String(extraDiskSize, defaultDiskSize, "Disk size allocated for extra disks attached to the minikube VM (format: <number>[<unit>], where unit = b, k, m or g).")
```

Comment:
add ` only used by hyperkit driver`

Reply:
I opted to message this as `currently only implemented for hyperkit driver` since I think that makes it more clear that it can be implemented for other drivers but isn't currently. Let me know if your wording is still preferred.